media-sound/teamspeak-server-bin/Manifest

```diff
@@ -1,4 +1,6 @@
 DIST teamspeak3-server_linux_amd64-3.0.13.6.tar.bz2 6492488 BLAKE2B 4cad8545968423c2e65853d73f99a46cbccce628982c0a4418eaf34802d6a171f7e89a4ffc2fbd9fd82afdd9ffcd58d214c894ad88862c2927f1aec72f2d3087 SHA512 95271e5a44c7a14b8fbe134b855973c1af53ac76e6ba01089116b1f63c03bae25f5abfd176f9e26207aa4da8501d646298128078ab5b475458a7ccaba2a48e9c
 DIST teamspeak3-server_linux_amd64-3.0.13.8.tar.bz2 6916065 BLAKE2B 0ff4d86a03c910739f27b624d023ba3a5f323f4d56bca6a29a5ce8a737901372ff22cf18f2cc589af07b373fd17f952896ae2aa623571fc004a653125772aa0b SHA512 d8985f5850602660a6c037c1768a97c79b5842640d618ad00d242035a12820b52ba959545f3e668eb3d6079b1a228ff648c569e7d1fc64be6743ee77995ab0be
+DIST teamspeak3-server_linux_amd64-3.1.1.tar.bz2 7798155 BLAKE2B 4962968b33eaaa07043cfaa23d020d3d7dcff19b9e380f797196d382a4d004fb0ca2a56246eedf246f796e1bea8f719e80e715b2eda67e5340b7042dd1111a57 SHA512 8561cbeb16a44889ddaa6ab054b47abaaa6a5c942f39c308f6afd2687110bc9e142a8711afe557fa22fe92d1395b844a0eff8c8043921aee4e8070571bee6e44
```

Comment:
It seems you can't download this version anymore. Could you update, please?

Reply:
Are you sure? I was able to download it?
Anyway, I've dropped RESTRICT="fetch", as I don't see, that the license would deny this.

Reply:
Heh, I missed SRC_URI ;-).